### PR TITLE
Fix Formatting

### PR DESCRIPTION
**Pull request by luiscarlin**
This PR:
- adds prettier to the project
- formats all the files in the project

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["esbenp.prettier-vscode"]
```
Comment (luiscarlin) [inline .vscode/extensions.json:2]: this will tell vscode to recommend you to install this extension

Comment (luiscarlin) [reply .vscode/extensions.json:2]: this vscode extension runs prettier

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true
+}
```
Comment (luiscarlin) [inline .vscode/settings.json:4]: this tells vscode to use this extensions as the default formatter and to format a file when you save it

File: README.md [hunk 3]
```diff
@@ -38,4 +38,7 @@ npx playwright test --headed
 ```bash
 # display results
 npx playwright show-report
+
+# run formatter manually
+npm run format
```
Comment (luiscarlin) [inline README.md:43]: added this in the list of commands in the readme

File: package.json [hunk 4]
```diff
@@ -3,12 +3,16 @@
   "version": "1.0.0",
   "description": "Playwright UI automation tests for https://www.automationexercise.com/",
   "main": "index.js",
-  "scripts": {},
+  "scripts": {
+    "format": "prettier --write ."
```
Comment (luiscarlin) [inline package.json:7]: adds the command `npm run format` so we can run prettier manually on all files of the repo, which is what I did for this PR

File: package.json [hunk 5]
```diff
@@ -3,12 +3,16 @@
...
+  },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "devDependencies": {
     "@playwright/test": "^1.40.1",
-    "@types/node": "^20.11.0"
+    "@types/node": "^20.11.0",
+    "@types/prettier": "^3.0.0",
+    "prettier": "^3.2.2"
```
Comment (luiscarlin) [inline package.json:16]: This was added here after I ran: 

```bash
npm install --save-dev prettier @types/prettier
```

was necessary for us to be able to run prettier manually (not through the vscode extension)

**Comment by gcarlin01** [comment]
Nice! thanks for adding the extension recommendation and the new script command 😃👍🏼

**Pull request merged**